Doc/using/cmdline.rst

```diff
@@ -975,3 +975,13 @@ Debug-mode variables
    shutting down the interpreter.
 
    Need Python configured with the :option:`--with-trace-refs` build option.
+
+.. envvar:: PYTHONDUMPREFSFILE
+
+   If set, Python will dump objects and reference counts still alive
+   as file named `$PYTHONDUMPREFSFILE` still alive after shutting down
+   the interpreter.
```

Comment:
```suggestion
   If set, Python will dump objects and reference counts still alive
   after shutting down the interpreter into a file called `$PYTHONDUMPREFSFILE`.
```

Is it possible to use ``.. envvar:: PYTHONDUMPREFSFILE=FILENAME`` syntax and then use ``FILENAME`` in the text?